astropy/stats/funcs.py

```diff
@@ -1019,6 +1019,77 @@ def biweight_midvariance(a, c=9.0, M=None, axis=None):
 
     return (n ** 0.5) * f1 / f2
 
+def biweight_midcovariance(a, c=9.0, M=None):
+    r"""
+    Compute the biweight midcovariance.
+
+    This is a robust and resistant estimator of the covariance matrix.
+
+    For more details, see:
+    `<http://www.itl.nist.gov/div898/software/
+    dataplot/refman2/auxillar/biwmidc.htm>_.`
```

Comment:
This link cannot be resolved in this form by the docs builder and thus raises a failure in travis. 

Also, I suggest to move the link to the bottom of the docstring under a new ``References`` section. You can find examples for formatting that in the docsstring of other stats functions (e.g. jackknife or lombscargle)